src/index.tsx

```diff
@@ -134,7 +134,8 @@ const DateBlock: React.FC<DateBlockProps> = ({
     }, [scrollRef.current])
 
     const handleChange = ({nativeEvent}: any) => {
-        const digit = nativeEvent.contentOffset.y / dHeight + digits[0];
+        let digit = nativeEvent.contentOffset.y / dHeight + digits[0];
+        digit = parseInt(digit.toFixed(0), 10);
 
         if (Number.isInteger(digit)) {
             onChange(type, digit);
```

Comment:
I think this block is no longer needed.

Reply:
@jorisdugue what about this check?

Reply:
@DieTime This verification is no longer really necessary because we transform our number into a basic interger

Reply:
After another check it can happen that the result is NaN this allows not to update the information (especially at the first rendering)